tests/installation/livecdreboot.pm

```diff
@@ -81,9 +82,10 @@ sub run() {
             save_screenshot;
         }
         if (get_var("XEN")) {
+            send_key 'up';    # prevent grub2 timeout
```

Comment:
ah, I see :)
anyway I already moved between line 73 and line 74, let's see how this work on openqa.o.o, I don't think it will breaks normal boot though.